Gems/Atom/RHI/Metal/CMakeLists.txt

```diff
@@ -11,3 +11,5 @@ set(gem_json ${gem_path}/gem.json)
 o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
 
 add_subdirectory(Code)
+
+ly_create_alias(NAME Atom_RHI_Metal.HeadlessServers NAMESPACE Gem)
```

Comment:
repeat: This should be added in the `Gems/Atom/RHI/Metal/Code/CMakeLists.txt` and grouped together with the only ly_create_alias calls